pyodide_build/pypabuild.py

```diff
@@ -242,6 +242,23 @@ def make_command_wrapper_symlinks(symlink_dir: Path) -> dict[str, str]:
     return env
 
 
+@contextmanager
+def _get_symlink_dir(env: dict[str, str], build_dir: Path | None):
+    if build_dir:
+        # If we're running under build-recipes, leave the symlinks in
+        # the build directory. This helps with reproducing.
+        symlink_dir = build_dir / "symlinks"
+        shutil.rmtree(symlink_dir, ignore_errors=True)
+        symlink_dir.mkdir()
+        yield symlink_dir
+        return
```

Comment:
Considering that this function deletes and recreates the build directory. The name `_get_symlink_dir` is somehow misleading to me. How about renaming the function such as `_create_symlink_dir`?